src/pages/blog/index.astro

```diff
@@ -13,50 +13,45 @@ export async function getPosts() {
   return await useGetPostsQuery(true)
 }
 
-const tags = await getTags()
-
 const posts = await getPosts()
+console.log(posts)
 ---
 
 <Layout title={SITE_TITLE} description={SITE_DESCRIPTION}>
-  <section class="flex flex-col">
-    <h1>Blog</h1>
-    <div class="flex flex-col">
-      <div>
-        <input type="text" placeholder="Search blog" />
-      </div>
-      <div>
-        <ul class="flex">
-          {
-            tags.map(({ slug, title }) => (
-              <li class="p-2">
-                <a href={slug}>#{title}</a>
-              </li>
-            ))
-          }
-        </ul>
-      </div>
-    </div>
-  </section>
-  <section class="xxl:grid-cols-4 grid grid-flow-row grid-cols-1 gap-4 py-24 md:grid-cols-2 md:gap-8 xl:grid-cols-3">
+  <SectionContainer className="my-24" containerClass="flex flex-col gap-20">
+    <h1 class="text-4xl font-medium uppercase text-gray md:text-5xl lg:text-6xl">Articles and news</h1>
+    <input class="max-w-sm" type="text" placeholder="Search blog" />
+
     {
-      posts.map(({ featuredImage, title, pubDate, slug, estimatedReadingTime }) => (
-        <div class="flex flex-col">
-          <div class="relative h-[208px] overflow-hidden rounded-md">
-            {featuredImage && <FeaturedImage class="max-h-52" width={360} height={240} asset={featuredImage.asset.url} alt={title} />}
+      posts.map(({ featuredImage, title, slug, description }) => (
+        <div class="flex w-full flex-col gap-12 md:flex-row">
+          <div class="w-full md:w-1/2 lg:w-2/5">
+            {featuredImage && (
+              <FeaturedImage class="aspect-square h-full max-h-[420px] w-full max-w-[550px]" asset={featuredImage.asset.url} alt={title} />
```

Comment:
No `max-h[xxx]` and `max-w-[xxx]`. Please, make this dynamically fit to content.